src/main/kotlin/org/ptss/support/domain/interfaces/controllers/IInvitationController.kt

```diff
@@ -0,0 +1,72 @@
+package org.ptss.support.domain.interfaces.controllers
+
+import jakarta.ws.rs.Consumes
+import jakarta.ws.rs.POST
+import jakarta.ws.rs.Path
+import jakarta.ws.rs.Produces
+import jakarta.ws.rs.core.MediaType
+import jakarta.ws.rs.core.Response
+import org.eclipse.microprofile.openapi.annotations.Operation
+import org.eclipse.microprofile.openapi.annotations.responses.APIResponse
+import org.eclipse.microprofile.openapi.annotations.responses.APIResponses
+import org.ptss.support.api.dtos.requests.invitations.UserInvitationRequest
+import org.ptss.support.api.dtos.requests.invitations.UserInvitationVerificationRequest
+import org.ptss.support.api.dtos.requests.invitations.UserRegistrationRequest
+
+@Path("/users")
+@Produces(MediaType.APPLICATION_JSON)
+@Consumes(MediaType.APPLICATION_JSON)
+interface IInvitationController {
+    @POST
+    @Path("/invite")
+    @Operation(summary = "Invite user to join group")
+    @APIResponses(
+        APIResponse(
+            responseCode = "201",
+            description = "Invitation sent successfully"
```

Comment:
miss
```suggestion
            description = "Invitation created & sent successfully"
```
ivm het feit dat je in db invitation code stuurt en email verstuurd direct erna